packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -1,6 +1,6 @@
-import { HighContrastSelector, AnimationVariables, normalize, IStyle } from '../../Styling';
+import { HighContrastSelector, AnimationVariables, normalize, IStyle, getPlaceholderStyles, getGlobalClassNames } from '../../Styling';
 import { ISearchBoxStyleProps, ISearchBoxStyles } from './SearchBox.types';
-import { getPlaceholderStyles, getGlobalClassNames } from '../../Styling';
+import { getRTL } from '@uifabric/utilities';
```

Comment:
To follow the pattern of every other imports from utilities package, internally we usually import from `../../Utilities`. Depending how many levels deep is the component the preceding `../` can change.